etc/spack/modules.yaml

```diff
@@ -5,4 +5,14 @@
 # although users can override these settings in their ~/.spack/modules.yaml.
 # -------------------------------------------------------------------------
 modules:
+  prefix_inspections: {
+        bin: ['PATH'],
```

Comment:
Oh I see -- the whole thing is in braces.  I don't think you have to do that if the dict is spread over multiple lines.